hera_mc/utils.py

```diff
@@ -26,6 +26,47 @@ def str_to_bytes(s):
     def bytes_to_str(b):
         return b.decode('utf8')
 
+from astropy.time import Time
+from astropy.time import TimeDelta
+from astropy import coordinates as coord
+from astropy import units as u
+import numpy as np
+import datetime
+
+def LSTScheduler(starttime,LSTbin_size):
+    #input an astropy time object
+    #input an LSTbin_size in seconds
+    #   grid the day into regular LST bins of size LSTbin_size
+    #   return the time of the next occuring LST bin
+
+    sidesec = 365.25/366.25 #length of sidereal second in SI seconds.
```

Comment:
I'm worried about hardcoding this, can't we get it from astropy?

Reply:
how about
sidesec = Quantity(1,'sday').to('day').value  #seems to be close at 1e-8